Package.resolved

```diff
@@ -132,8 +132,8 @@
         "repositoryURL": "https://github.com/stencilproject/Stencil.git",
         "state": {
           "branch": null,
-          "revision": "94197b3adbbf926348ad8765476a158aa4e54f8a",
```

Comment:
I've seen this bumped in another PR too. I'd remove this from this PR and open a separate PR for the version bump. That way we can capture the changes in the CHANGELOG and make it easier to revert in case it causes issues.

Reply:
Yeah, sometimes SPM updates the packages just by you opening a project via Xcode (or maybe Xcode does it). It's quite annoying

Reply:
For sure, i think this was Xcode. Will undo 👍🏼